### PR TITLE
 added function def and docstring

### PR DESCRIPTION
**Pull request by ManishPJoshi**
Can you please approve and merge the request. 
Thanks

### CODE REVIEW

File: pypuck/pypuck.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+def season_stats(start_season=None, end_season = None):
+    """
+    Get a seasons stats specified by start year or start year and end year.
+    If no year is specified then the year 2019-2020 is default.
+    If an end year is specified then the start year is also to be provided.
+    year is to be provided in a 2 year format of YYYYYYYY 
+	
+    Parameters
+    ----------
+
```
Comment (jnederlo) [inline pypuck/pypuck.py:14]: Remove Newline

File: pypuck/pypuck.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	start_season : str
+	  The stat start year string in 'YYYYYYYY' format.
+	end_date : str
+	  The stat end year string in 'YYYYYYYY' format.
+    Returns
```
Comment (jnederlo) [inline pypuck/pypuck.py]: Add Newline

File: pypuck/pypuck.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    -------
+    pandas.core.DataFrame
+      The season stats in a dataframe.
+    Examples
```
Comment (jnederlo) [inline pypuck/pypuck.py]: Add Newline

File: pypuck/pypuck.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    If an end year is specified then the start year is also to be provided.
+    year is to be provided in a 2 year format of YYYYYYYY 
+	
+    Parameters
+    ----------
+
+	start_season : str
+	  The stat start year string in 'YYYYYYYY' format.
+	end_date : str
+	  The stat end year string in 'YYYYYYYY' format.
```
Comment (jnederlo) [inline pypuck/pypuck.py]: Add Newline

File: pypuck/pypuck.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    Returns
+    -------
+    pandas.core.DataFrame
+      The season stats in a dataframe.
```
Comment (jnederlo) [inline pypuck/pypuck.py]: Add Newline

**Review by jnederlo — CHANGES_REQUESTED** [review]
Looks good, but please make the small formatting changes.

File: pypuck/pypuck.py (hunk 5 above)
Comment (ManishPJoshi) [reply pypuck/pypuck.py]: done. I have added the new lines as suggested

Comment (ManishPJoshi) [reply pypuck/pypuck.py]: staggered the indentations

**Review by jnederlo — APPROVED** [review]
Changes look good!. Approved.

**Pull request merged**